php/class-fm-bylines-cli.php

```diff
@@ -0,0 +1,134 @@
+<?php
+WP_CLI::add_command( 'fm-bylines', 'FM_Bylines_CLI' );
+
+class FM_Bylines_CLI extends WP_CLI_Command {
+
+	/**
+	 * Migrate Co-Authors Plus Guest Authors and linked accounts to FM Bylines
+	 *
+	 */
+	public function migrate_coauthors() {
+		global $coauthors_plus;
+
+		$cap_terms = get_terms( $coauthors_plus->coauthor_taxonomy, array( 'hide_empty' => false ) );
```

Comment:
I'm not sure what you mean by fail if not enabled. This doesn't require to have cap enabled. Just that there is data. If you want me to stick in an error message if cap_terms is empty I can do that.